pkg/webui/console/components/device-data-form/messages.js

```diff
@@ -29,26 +38,33 @@ export default defineMessages({
   deviceDescDescription:
     'Optional device description; can also be used to save notes about the device',
   deviceDescPlaceholder: 'Description for my new device',
-  deviceEUIDescription: 'The DevEUI is the unique identifier for this device on the network',
+  deviceEUIDescription: 'The DevEUI is the unique identifier for this device',
   deviceIdPlaceholder: 'my-new-device',
   deviceNamePlaceholder: 'My New Device',
   external: 'External',
   externalJoinServer: 'External Join Server',
-  fwdNtwkKeyDescription: 'Forwarding Network Session Integrity Key (or LoRaWAN 1.0.x NwkSKey)',
-  joinEUIDescription: 'JoinEUI identifies the Join Server (in LoRaWAN 1.0.x known as AppEUI)',
+  fNwkSIntKeyDescription: 'Forwarding Network Session Integrity Key',
+  homeNetID: 'Home NetID',
+  joinEUIDescription: 'JoinEUI identifies the Join Server',
   leaveBlankPlaceholder: 'Leave blank to generate automatically',
-  ntwkSEncKeyDescription: 'Network Session Encryption Key (only for LoRaWAN 1.1+)',
+  multicast: 'Multicast',
+  nsServerKekLabel: 'Network Server KEK Label',
+  nsServerKekLabelDescription:
+    'The KEK label of the Network Server to use for wrapping the network session key',
+  ntwkSEncKeyDescription: 'Network Session Encryption Key',
```

Comment:
Can we use the official nwk instead of ntwk? Also in other places

Reply:
See my comment above